scripts/configs/active_sampler_learning.yaml

```diff
@@ -1,4 +1,80 @@
 # Active sampler learning experiments.
+
+# Experiment with random explorer, myopic classifier in bumpy_cover.
+---
+APPROACHES:
+  random-explore:
+    NAME: "active_sampler_learning"
+    FLAGS:
+      explorer: "random_nsrts"
+ENVS:
+  bumpy_cover:
+    NAME: "bumpy_cover"
+ARGS:
+  - "debug"
+FLAGS:
+  strips_learner: "oracle"
+  sampler_learner: "oracle"
+  bilevel_plan_without_sim: "True"
+  max_initial_demos: 0
+  num_train_tasks: 1000
+  num_test_tasks: 100
+  max_num_steps_interaction_request: 50
+  sampler_mlp_classifier_max_itr: 100000
+  mlp_classifier_balance_data: False
+  pytorch_train_print_every: 10000
+  active_sampler_learning_use_teacher: False  # NOTE
+  online_nsrt_learning_requests_per_cycle: 10  # NOTE
```

Comment:
What are the two `NOTE` comments here for?

Reply:
these are just things that change meaningfully between experiments and might be easy to miss